test/FunctionalTests/Grpc.AspNetCore.FunctionalTests.csproj

```diff
@@ -5,7 +5,7 @@
 
     <IsPackable>false</IsPackable>
 
-    <LangVersion>latest</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
For some reason "latest" is defaulting to 7.3 (?) of csharp and not 8.0. This might be a preview thing. I think this is required for now and can change to latest in the future.

8.0 is required for static local methods.